boards/raspberry-pi-4/ubuntu_server_20.04_arm64.json

```diff
@@ -38,7 +38,14 @@
     {
       "type": "shell",
       "inline": [
-        "touch /tmp/test"
+        "mv /etc/resolv.conf /etc/resolv.conf.bk",
```

Comment:
If this is going to be added I would do it as a seperate provisioner. I'd also suggest using Ansible rather than a shell script. However overall I think this pollutes the board image generation and should be done via optionally calling into another builder.

Reply:
Depends... ansible is good at idempotent playbooks, that are re-run to update a machine... the update case does not exist here... that is why also ansible/puppet/chef are not used inside a Dockerfile... the packer configs here also run always end2end similar to a Dockerfile ... not need to bring in a ansible..

Another way would be extracting the added code here to e.g. `./scripts/setup-docker-ubuntu.sh` ... and use it from there inside a provisioner... similar to the resize scripts already in the scripts folder.. this would make it a bit more reusable...